radio/src/targets/common/arm/stm32/usb_driver.cpp

```diff
@@ -181,36 +184,28 @@ void usbJoystickUpdate()
 #endif
 
     // 4 axes
-    for (int i = 0; i < 8; ++i) {
-      int16_t value = channelOutputs[i] + 1024;
-      if ( value > 2047 ) value = 2047;
-      else if ( value < 0 ) value = 0;
-      HID_Buffer[i*2 + 0] = uint8_t(value);
-      HID_Buffer[i*2 + 1] = uint8_t(value >> 8);
-    }
-    // 2 sliders
-    // 4 switches
+    HID_Buffer[0] = uint8_t(adcValues[STICK1] >> 4) - 0x80;
```

Comment:
This changes "standard" EdgeTX/OpenTX behaviour to use values modified by mixers, please revert it.

Reply:
See my comment above. Current implementation is plain broken in any sense. However I'll keep it and add another flag to the build.